.yamato/pack-autodesk-fbx.yml

```diff
@@ -0,0 +1,101 @@
+# This yml file contains jobs for packing com.autodesk.fbx package which is a submodule of com.unity.formats.fbx
+
+{% metadata_file .yamato/global.metafile %}
+---
+
+build_win:
+  name: Build on win
+  agent:
+    type: {{ win_platform.type }}
+    image: {{ win_platform.image }}
+    flavor: {{ win_platform.flavor}}
+  commands:
+    # Load submodule and update it
+    - git submodule update --init --recursive
+    - git submodule update --remote
+    - choco -v -y install 7zip
+    - python External\com.autodesk.fbx\build.py --stevedore --verbose --clean --yamato
+    - |
+      cd External\com.autodesk.fbx
+      ren build build-win
+  artifacts:
+    build:
+      paths:
+        - "External/com.autodesk.fbx/build-win/install/**"
+
+build_mac:
+  name: Build on mac
+  agent:
+    type: {{ mac_platform.type }}
+    image: {{ mac_platform.image }}
+    flavor: {{ mac_platform.flavor}}
+  variables:
+    HOMEBREW_NO_INSTALL_CLEANUP: "1"
+  commands:
+    # Load submodule and update it
+    - git submodule update --init --recursive
+    - git submodule update --remote
+    # We need to install p7zip.
+    - brew list p7zip || brew install p7zip
+    - cmake --version || brew install cmake
+    - python ./External/com.autodesk.fbx/build.py --stevedore --verbose --clean --yamato
+    - mv ./External/com.autodesk.fbx/build ./External/com.autodesk.fbx/build-mac
+  artifacts:
+    build:
+      paths:
+        - "External/com.autodesk.fbx/build-mac/install/**"
+
+build_ubuntu:
```

Comment:
Good suggestion. But unfortunately `metafile` won't work because yamato files inside submodule won't be seeable to the host repo which is `com.unity.formats.fbx`.
.cmd or .sh files may work. I will give a try.

Reply:
@vkovec Please re-review my PR. In `com.autodesk.fbx` package, I have tried putting build commands on different platform into three build scripts `build_win.cmd`, `build_mac.sh` and `build_linux.sh` respectively. The changes are only in a [feature branch](https://github.com/Unity-Technologies/com.autodesk.fbx/tree/wind/put_build_commands_in_script_files) now. 
In order to test whether it works with `com.unity.formats.fbx,` I specified the feature branch in submodule file `.gitmodules` as below:
```
[submodule "External/com.autodesk.fbx"]
	path = External/com.autodesk.fbx
	url = https://github.com/Unity-Technologies/com.autodesk.fbx.git
	branch = wind/put_build_commands_in_script_files
```
And it seems it is working well.
If everything looks good to you, I will create a PR for `com.autodesk.fbx` package and merge the changes there first and then make submodule point to master branch.